docs/sources/administration/provisioning.md

```diff
@@ -157,11 +157,12 @@ Since not all datasources have the same configuration settings we only have the
 | interval                | string  | Elasticsearch                                                    | Index date time format. nil(No Pattern), 'Hourly', 'Daily', 'Weekly', 'Monthly' or 'Yearly' |
 | logMessageField         | string  | Elasticsearch                                                    | Which field should be used as the log message                                               |
 | logLevelField           | string  | Elasticsearch                                                    | Which field should be used to indicate the priority of the log message                      |
-| authType                | string  | Cloudwatch                                                       | Auth provider. keys/credentials/arn                                                         |
-| assumeRoleArn           | string  | Cloudwatch                                                       | ARN of Assume Role                                                                          |
-| defaultRegion           | string  | Cloudwatch                                                       | AWS region                                                                                  |
+| authType                | string  | Cloudwatch                                                       | Auth provider. default/credentials/keys                                                     |
+| externalId              | string  | Cloudwatch                                                       | Optional External ID                                                                        |
+| assumeRoleArn           | string  | Cloudwatch                                                       | Optional ARN role to assume                                                                 |
+| defaultRegion           | string  | Cloudwatch                                                       | Optional default AWS region                                                                 |
```

Comment:
Should these be in alphabetical order?

Reply:
@papagian I have no idea. Do we have any convention for it?

Reply:
I thought that the table was [alphabetically ordered](https://grafana.com/docs/grafana/latest/administration/provisioning/#json-data) (by datasource) but I see some inconsistencies so never mind...